willow/lib/sufia/configuration.rb

```diff
@@ -0,0 +1,159 @@
+module Sufia
+  class Configuration
+    def curation_concerns_config
+      @curation_concerns_config ||= CurationConcerns.config
+    end
+
+    delegate(*(CurationConcerns.config.methods - Object.methods),
+             to: :curation_concerns_config)
+
+    attr_writer :banner_image
+    def banner_image
+      @banner_image ||= "https://cloud.githubusercontent.com/assets/4950775/24611147/02ee9496-1879-11e7-89f4-ec17e1e7dfa7.jpg"
```

Comment:
I struggled to make the assets path work relative to the config - the original config also had the file hosted somewhere else, so I followed suit. I can see if I can't change that if it is the better thing to do.